flowzone.yml

```diff
@@ -1513,6 +1518,11 @@ jobs:
       fail-fast: false
       matrix: ${{ fromJSON(needs.docker_check.outputs.docker_bake_matrix) }}
 
+    # remove all permissions from the default GITHUB_TOKEN
+    # https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token
+    # https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#permissions
+    permissions: {}
```

Comment:
duplicated permission logic at the job level in case we restore some permissions at the workflow level

just an extra safeguard against leaking secrets with actual permissions